src/layout/components/DiscordPage/index.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import Iframe from 'react-iframe';
+import ThemeContext from '../../../styles/ThemeContext/themeContext';
 
 import './style.less';
 
 const DiscordPage = () => {
+  const { theme, toggleTheme } = React.useContext(ThemeContext);
   const ReactIframe = Iframe as React.ComponentClass<any>;
   return (
     <div className="discord-page">
       <h1 className="title">Discord</h1>
       <ReactIframe
-        url="https://discordapp.com/widget?id=573028991527550986&theme=light"
+        url={"https://discordapp.com/widget?id=573028991527550986&theme=" + theme}
```

Comment:
can template literal this, eg
`` `https://discordapp.com/widget?id=573028991527550986&theme=${theme}` ``